tests/test_newsheadlines.py

```diff
@@ -0,0 +1,28 @@
+import unittest
+
+import requests
+
+from app.newsheadlines import NewsHeadlines
+
+class TestNews(unittest.TestCase):
+    def setUp(self):
+        self.news = NewsHeadlines()
+
+    def test_api_key_collection(self):
+        self.assertEqual(self.news.get_key(), '9b40cf73093e47e586cd61132f1d56fe')
+    
+    def test_cli_function(self):
+        api_key = self.news.get_key()
+        url = 'https://newsapi.org/v2/top-headlines?sources=fox-news&apiKey='+api_key+'&pageSize=10'
+        news_sources = requests.get(url).json()['articles']
```

Comment:
It's bad to make API requests when testing API consumption.
A better approach would be creating a mock response and making a mock request and checking whether the correct mock response was returned